backend/eduschola_project/edusch_app/models.py

```diff
@@ -21,8 +21,8 @@ class Course(models.Model):
     course_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     name = models.CharField(max_length=255)
     description = models.TextField()
-    teacher = models.ForeignKey('User', on_delete=models.CASCADE)  # Many-to-One relationship with User model (Teacher)
-    school = models.ForeignKey(School, on_delete=models.CASCADE)  # Many-to-One relationship with School model
+    teacher = models.ForeignKey('User', on_delete=models.SET_NULL, null=True)  # Many-to-One relationship with User model (Teacher)
```

Comment:
so you can create a course without a teacher and school existing ?

Reply:
No. Not possible

Reply:
so what is the new cascading styleto SET_NULL means?

Reply:
If you delete a staff, It does not delete the user details, it should not delete the course